web/src/components/DatabaseConfigForm.tsx

```diff
@@ -56,16 +62,16 @@ export const DatabaseConfigForm = ({
           setValue={setPassword}
           type="password"
         />
-        {showDatabase && (
-          <ConfigInput
-            label="Database"
-            placeholder="martech"
-            value={database}
-            setValue={setDatabase}
-          />
-        )}
-        {showScaleFactor && <ScaleFactorSelector />}
       </SimpleGrid>
+      {showDatabase && (
```

Comment:
Why do we need `showDatabase` if it's always `true`?

Reply:
Its not always true

Its passed as parameter if you wan't to show database input box

Its shown in home page but not in config section